components/Layout/Footer.tsx

```diff
@@ -0,0 +1,20 @@
+// Libraries
+import React from "react";
+
+// Styles
+import styles from "./Layout.module.css";
+
+function Footer() {
+  console.log("Load Footer");
+  return (
+    <footer className={styles.footer}>
+      <a>A website for browsing movies.</a>
+
+      <a href="https://github.com/ericc0429">
+        Created by Eric Chen (ericc0429).
+      </a>
+    </footer>
+  ); // End return
+}
+
+export default React.memo(Footer);
```

Comment:
You can do this but this component is so small and simple almost not worth it. You might add more over had then performance gains. Good write up: 

https://felixgerschau.com/react-performance-react-memo/